index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index2.css" />
+  </head>
+  <body>
```

Comment:
- [ ] [REQUIRED] Please, use a `<header>` tag for the navbar and logo section. to follow semantic rules.
- [ ] [REQUIRED] Please, make sure to wrap the intro (hero) section in a `<section>` element 🎁. for semantic code base.

### Demonstration

![sections](https://user-images.githubusercontent.com/60979926/164576959-b4326b1e-22d9-4c48-a655-5adc59ba8fda.png)